core/Cargo.toml

```diff
@@ -18,14 +18,23 @@ categories = ["no-std", "embedded"]
 include = ["/Cargo.toml", "src/**/*.rs", "/README.md", "/LICENSE"]
 
 [dependencies]
-ink_alloc = { path = "../alloc/" }
-ink_utils = { path = "../utils/" }
-parity-codec = { version = "3.2", default-features = false, features = ["derive", "full"] }
+ink_abi = { path = "../abi", default-features = false }
```

Comment:
abi not in master yet

Reply:
thank god you found it, I was already suspicious about the CI error ...